README.md

```diff
@@ -804,16 +811,17 @@ You can specify User Groups within a TIB Profile. This can either be a static or
 
 ```
 {
-  "DefaultUserGroupID": "default-user-group",
+  "DefaultUserGroupID": "<dashboard-user-group-id>",
   "CustomUserGroupField": "scope",
   "UserGroupMapping": {
     "admin": "<admin-group-id>",
     "analytics": "<analytics-group-id>"
   }
 }
 ```
-* For a static setting, use `DefaultUserGroupID`
-* For a dynamic setting based on OAuth/OpenID scope, use `CustomUserGroupField` with  `UserGroupMapping` listing your User Groups and ID in the following format - `"<User Group Name>": "<ID>".
+When doing SSO for a user, you need to think about the user's permissions once they are logged into the application. During the SSO flow of a user, TIB can request Tyk-Dashboard to login that user with certain user group permissions. In order to configure the user's permission you need to create a group in the Dashboard and use this group object ID as a value for these fields: 
+* For a static setting and set `DefaultUserGroupID` with a Dashboard group id. TIB will use it as the default user permissions when requesting a nonce from the dashboard. **Note:** If you don't set this field, the user will be logged in as an admin dashboard user.
```

Comment:
Drop the **and** in: "For a static setting **and** set `DefaultUserGroupID` with a Dashboard group id"